app/src/main/java/org/android/go/sopt/util/Constants.kt

```diff
@@ -14,8 +14,8 @@ object Constants {
     const val NULL_JOIN = "빈칸이 있습니다."
     const val INPUT_SUCCESS = "Id,Pwd good!"
 
-    const val ID_REGEX = ""
-    const val PW_REGEX = ""
+    const val ID_REGEX = """^[a-z0-9]*$"""
+    const val PW_REGEX = """^(?=.*[0-9])(?=.*[a-zA-Z])(?=.*[!@#$%^+\-=])(?=\S+$).*$"""
```

Comment:
```suggestion
    const val ID_REGEX = """^(?=.*[0-9])(?=.*[a-zA-Z])(?=\S+$).{$ID_COUNT_MIN,$ID_COUNT_MAX}$"""
    const val PW_REGEX = """^(?=.*[0-9])(?=.*[a-zA-Z])(?=.*[!@#$%^+\-=])(?=\S+$).{$PWD_COUNT_MIN,$PWD_COUNT_MAX}$"""
```
혹시 아이디에 소문자나 숫자 중 한 가지만 포함되어도 통과되고 있지 않나요? 
그리고 정규식으로 글자 수까지 검사하도록 구현하여 다른 코드를 줄일 수 있을 것 같습니다!

Reply:
정규식 수정할게요!